src/Scripts/nextjs/index.ts

```diff
@@ -51,6 +52,24 @@ const commonQuestions: INextjsCommonQuestions = {
 };
 
 const questions: INextjsQuestions = {
+	AddPlugin: [
+		{
+			choices: [
+				new inquirer.Separator(),
+				{
+					name: 'Styled Components',
+					value: 'styled'
```

Comment:
bu value'ları enum mu yapsak? Daha sonradan takibinin kolay olması için

Reply:
`PluginHelper.` dediginde geliyor `styled` ve `sass`, onu mu kastediyorsun? Burada nasil enum yapariz bilemedim.